Content.Client/Storage/Visualizers/CrateLabelVisualLayers.cs

```diff
@@ -0,0 +1,5 @@
+namespace Content.Client.Storage.Visualizers;
+public enum CrateLabelVisualLayers : byte
```

Comment:
change to

```cs
public enum PaperLabelVisuals : byte
{
    Layer,
    HasLabel,
    LabelType
}

public enum PaperLabelType : byte
{
    Paper,
    Bounty
}
```
and move it to shared not client

then since nothing actually uses this yet, edit `LabelSystem` so when the label is added/removed/swapped uses code like this:
```cs
private void UpdateAppearance(Entity<PaperLabelComponent, AppearanceComponent?> ent)
{
    if (!Resolve(ent, ref ent.Comp2))
        return;
        
    var slot = ent.Comp1.LabelSlot;
    _appearance.SetData(ent, PaperLabelVisuals.HasLabel, slot.HasItem);
    if (slot.Item)
        _appearance.SetData(uid, PaperLabelVisuals.LabelType, PaperLabelType.Paper); // TODO: store label type on a component on the paper
```

and update ComponentInit/OnContainerModified to call `UpdateAppearance((ent, label));` instead of trygetcomponent + setdata

later ill implement the LabelType bit to support bounty sprite

since `PaperLabelVisuals` now has a layer you can also remove `BodyBagVisualLayers`' `Label` layer, did you know body bags had a sprite for when you put paper on them, and that it can be done at all?